### ISSUE TITLE
Deleted items overview

### ISSUE THREAD

**Issue by b-creative** [sachatrauwaen/OpenContent#141]
You know what would be a great enhancement... a deleted items overview. Very regularly I receive the wish of my clients to be able to recover the stuff they delete... a simple 'Are you sure...' is often not enough.

**Comment by Timo-Breumelhof** [comment]
Yes this has been discussed before, but obviously not implemented.
We (40FINGERS) are sponsoring https://github.com/sachatrauwaen/OpenContent/issues/131 as that leads to issues too in ML environments (users don't realize they delete an item in all languages). Although I wonder if users actually read those popups..

**Comment by b-creative** [comment]
Maybe an idea to change this into this new functionality? We also are willing to put in some sponsering...

**Comment by Timo-Breumelhof** [comment]
#131 is already implemented :-)

**Comment by b-creative** [comment]
@Timo-Breumelhof, I meant my suggestion... :)

**Comment by Timo-Breumelhof** [comment]
@b-creative right, we have a meeting about the issues we sponsor today, I'll mention this one too.
